addons/website_sale/views/sale_order_views.xml

```diff
@@ -53,6 +76,26 @@
         <field name="help">You don't have any order from the website.</field>
     </record>
 
+    <!-- Server action to send multiple recovery email-->
+    <record id="send_recovery_email_action_server" model="ir.actions.server">
+        <field name="name">Send a Recovery Email</field>
```

Comment:
As for mail templates we put the main business model in the server action name. That way when viewing the server action list they are ordered by name and therefore by main model. Here it would be Sale: Send Cart Recovery Email or something like that.